sycl/plugins/level_zero/pi_level0.cpp

```diff
@@ -1753,6 +1753,7 @@ pi_result piProgramLink(pi_context Context, pi_uint32 NumDevices,
                         const pi_program *InputPrograms,
                         void (*PFnNotify)(pi_program Program, void *UserData),
                         void *UserData, pi_program *RetProgram) {
+  assert("Program Linking is not supported yet in Level0" && false);
```

Comment:
please use `die` so that it terminates even when asserts are inactive,

Reply:
Done